contracts/upgradeable_contracts/ChaiConnector.sol

```diff
@@ -185,6 +185,9 @@ contract ChaiConnector is Ownable, ERC20Bridge, TokenSwapper {
 
         receiver.call(abi.encodeWithSelector(ON_TOKEN_TRANSFER, address(this), interest, ""));
 
+        // Additional constant is not needed here, since we allow to withdraw only extra part of chai balance,
+        // which is not necessary to cover 100% dai balance.
+        // It is guaranteed that the withdrawal of interest won't left the bridge balance uncovered.
```

Comment:
Do you have a unit test to cover possible scenarios to confirm this?

Reply:
Yes, it is covered by regular scenarios for `payInterest`. 

We are paying interest in Chai directly, without any conversion. So, we cannot lose anything there. When we are evaluating the amount of payed interest in Chai here: https://github.com/poanetwork/tokenbridge-contracts/blob/f45c0cf7aef7bb0cd2fada9be6ff9ecde71d9492/contracts/upgradeable_contracts/ChaiConnector.sol#L181 we leave the sufficient amount of chai on the bridge balance, which is enough to cover the whole xDai supply.

Reply:
> We are paying interest in Chai directly, without any conversion.
Yeah, OK. It makes sense.